static/templates/settings/organization_settings_admin.hbs

```diff
@@ -200,34 +200,9 @@
                     </div>
                     <div id="zoom_help_text" class="zoom_credentials">
                         <p>
-                            Note: Zoom support is experimental. In particular, Zulip currently supports having
-                            only one active Zoom meeting at a time.
+                            Note: Zoom support is experimental.
                         </p>
                     </div>
```

Comment:
I think we can probably remove this block entirely-- we'll likely contact existing Zoom users to make sure they test the new version, but once that's done, there's no reason we need to have an unusual warning against using it (which we did need when you could only do one call at a time).